tests/Behat/Resources/services/contexts/setup.yml

```diff
@@ -28,6 +28,9 @@ services:
         arguments:
```

Comment:
Why the `tests/Behat/Resources/services.xml` is an XML, but the current file is Yaml?

Reply:
It needs to stay that way as there is problem with paths in YAML, discussed with @senghe